bench/benchmarks/remove_paint_state_feature.js

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
The three benchmark tests here look to be almost identical with the exception of what's passed to `setFeatureState` and `removeFeatureState` in `bench`. You could probably create an overarching `remove_paint_state` benchmark that has sub-tests for each individual test, in the same vein as the Expressions and Layers benchmarks.

Reply:
@ryanhamley  was looking to do just that! thanks for the tip, was able to crib the format from Layers. consolidated the benches here https://github.com/mapbox/mapbox-gl-js/pull/7696/commits/d048d22d4a6f3224e811b31426333c78bdb2c718